README.md

```diff
@@ -128,7 +128,42 @@ $top10 = frequencyAnalysis($text)
     - `::of` (pure) and `->ap` (apply) with applicative laws
     - `->flatMap` (bind) with monadic laws
     - on top of that they support many friendly functional methods (like `exists`, `all`, `find` etc.) 
-- **TODO** (currying, lifting, traversing ...)
+
+#### Currying
```

Comment:
Because you split currying into separate section I suggest to change it to `###` level and rename `### Functional features` to something more subtle (Maybe `### Type classes` ?)

Reply:
I have changed headline level and renamed `Functional features` to `Type classes` for now.

Reply:
yup, we can rearrange